mlx/assets/traceability.js

```diff
@@ -1,4 +1,45 @@
-jQuery(function() {
+// item-tree
+jQuery(function () {
     $('ul.bonsai').bonsai();
 });
 
+
+// item
+$(document).ready(function () {
+    $('div.collapsible_links div.admonition:first-child').each(function (i) {
+        $(this).siblings('dl.simple').addCollapseButton($(this));
+    });
+});
+
+jQuery.fn.extend({
+    addCollapseButton: function (admonition) {
+        var relations = $(this);
+
+        if (relations.children().length > 0) {
+            if (admonition.parent().hasClass('collapse')) {
+                // collapse relations and attributes list for each item on page load
+                relations.toggle();
```

Comment:
Toggle here but not in else - Did you try the close/open/close/open cycle?

Reply:
This `toggle()` should only happen once (on page load) and only when the config variable has been set to `True` (which adds the 'collapse' class in Python code). It collapses the list of links without animation.

I have tried all the cycles and both configuration options. It works as intended. You can give it a go yourself to see if you want a change to the positioning, button scale, animation, ... :)